e2e/test/scenarios/permissions/download-permissions.cy.spec.js

```diff
@@ -95,7 +95,6 @@ describeEE("scenarios > admin > permissions > data > downloads", () => {
 
   it("respects 'no download' permissions when 'All users' group data permissions are set to `Blocked` (metabase#22408)", () => {
     cy.visit(`/admin/permissions/data/database/${SAMPLE_DB_ID}`);
-
```

Comment:
@npfitz great way to beef your stats

Reply:
Don't tell @iethree I'm coming for his crown